requirements.txt

```diff
@@ -1,3 +1,5 @@
-boto3==1.2.3
+boto3==1.3.1
 mock==1.3.0
 nose==1.3.7
+pytz==2016.4
+six==1.10.0
```

Comment:
More of a question for @garnaat, but since placebo is used as a library and not an application, what would you think about unpinning these requirements?  Or, alternatively, hook this repo up to requires.io (free!) to get notified and autogenerate PRs when new versions are released?